examples/skills_tuning/instructlab/table_manipulation.py

```
@@ -0,0 +1,639 @@
+# %%
+generated_data = generator.generate(seed_data)
+
+# %% [markdown]
+# ## 🔍 Step 4: Explore and Validate the Synthetically Generated Data
+#
+# Once the skill generation pipeline has been executed, the output is a set of **synthetically generated examples** — new context-question-response triples that follow the same structure as the seed data but are expanded and refined by the teacher model.
+#
+# Below is an example of one generated entry:
+
+# %%
+import random
```

Flake8 (7.2.0)

[error] redefinition of unused 'random' from line 34

(F811)